core/app/[locale]/(default)/account/[tab]/_components/account-status-provider.tsx

```diff
@@ -17,6 +26,7 @@ export const AccountStatusProvider = ({
   isPermanentBanner?: boolean;
 }) => {
   const [accountState, setAccountState] = useState<AccountState>({ status: 'idle', message: '' });
+  const elementRef = useRef<HTMLAnchorElement>(null);
```

Comment:
Since we're using `AccountStatusContext` it probably makes more sense to specify which element we basically refer to. What do you think about using more specific name like `tabRef` or `activeTabRef`?

Reply:
+ for `activeTabRef`

Reply:
@bc-alexsaiannyi @yurytut1993 fixed!